trace-C19/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -2,7 +2,7 @@
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
   <threadsafe>false</threadsafe>
   <sessions-enabled>true</sessions-enabled>
-  <runtime>java8</runtime>
+  <runtime>java11</runtime>
```

Comment:
I am curious as to why you changed this to java11 because I think the project is set to run with Java8 on app engine.  Did this run in cloud shell?